From 98b8f1c5d1b15e76f604afd473488f765b094d31 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Fri, 1 Aug 2025 07:34:18 +0200
Subject: [PATCH 14/55] base-files: set the default password to "openwifi"

Sets a default root password for development and testing environments, avoiding empty password on initial boot.

Signed-off-by: John Crispin <john@phrozen.org>
---
 package/base-files/files/etc/shadow | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/base-files/files/etc/shadow b/package/base-files/files/etc/shadow
index 39bdb9c90a..5b3f32ce64 100644
--- a/package/base-files/files/etc/shadow
+++ b/package/base-files/files/etc/shadow
@@ -1,4 +1,4 @@
-root:::0:99999:7:::
+root:$1$Ev771IIe$RHpqhCLW3U6c5tg9MrkQD/:18473:0:99999:7:::
 daemon:*:0:0:99999:7:::
 ftp:*:0:0:99999:7:::
 network:*:0:0:99999:7:::
-- 
2.34.1

